tests/integration/test_tiling.py

```diff
@@ -147,3 +183,60 @@ def test_val_dataloader(self, fxt_det_data_config) -> None:
 
     def test_tile_merge(self):
         pytest.skip("Not implemented yet")
+
+    def test_ov_det_tile_model(
+        self,
+        tmp_path: Path,
+        fxt_accelerator: str,
+        fxt_target_dataset_per_task: dict,
+    ):
+        tile_recipes = [recipe for recipe in pytest.RECIPE_LIST if "detection" in recipe and "tile" in recipe]
+        for tile_recipe in tile_recipes:
+            engine = Engine.from_config(
+                config_path=tile_recipe,
+                data_root=fxt_target_dataset_per_task[OTXTaskType.DETECTION.value.lower()],
+                work_dir=tmp_path / OTXTaskType.DETECTION,
+                device=fxt_accelerator,
+            )
+            engine.train(max_epochs=1)
+            exported_model_path = engine.export()
+            assert exported_model_path.exists()
+            engine.test(exported_model_path, accelerator="cpu")
+
+            ov_model = OVDetectionModel(model_name=exported_model_path, num_classes=3)
+
+            assert isinstance(ov_model.model, DetectionTiler), "Model should be an instance of DetectionTiler"
+            assert engine.datamodule.config.tile_config.tile_size[0] == ov_model.model.tile_size
+            assert engine.datamodule.config.tile_config.overlap == ov_model.model.tiles_overlap
+
+    def test_ov_inst_tile_model(
+        self,
+        tmp_path: Path,
+        fxt_accelerator: str,
+        fxt_target_dataset_per_task: dict,
+    ):
+        # Test that tiler is setup correctly for instance segmentation
+        tile_recipes = [
+            recipe for recipe in pytest.RECIPE_LIST if "instance_segmentation" in recipe and "tile" in recipe
+        ]
+
+        for tile_recipe in tile_recipes:
+            engine = Engine.from_config(
+                config_path=tile_recipe,
+                data_root=fxt_target_dataset_per_task[OTXTaskType.INSTANCE_SEGMENTATION.value.lower()],
+                work_dir=tmp_path / OTXTaskType.INSTANCE_SEGMENTATION,
+                device=fxt_accelerator,
+            )
+            engine.train(max_epochs=1)
+            exported_model_path = engine.export()
+            assert exported_model_path.exists()
+            engine.test(exported_model_path, accelerator="cpu")
+
+            ov_model = OVInstanceSegmentationModel(model_name=exported_model_path, num_classes=3)
+
+            assert isinstance(
+                ov_model.model,
+                InstanceSegmentationTiler,
+            ), "Model should be an instance of InstanceSegmentationTiler"
+            assert engine.datamodule.config.tile_config.tile_size[0] == ov_model.model.tile_size
+            assert engine.datamodule.config.tile_config.overlap == ov_model.model.tiles_overlap
```

Comment:
Are these overlapped with the existing integration tests?

Reply:
CLI tests currently cover tile recipes, however, engine doesn't cover those recipes and I added a new test in engine API test.